external/build/youtube-playlist.js

```diff
@@ -0,0 +1,181 @@
+/**
+ * @fileoverview Fetches the most recent playlists by specified channels from YouTube
+ */
+
+require('dotenv').config();
+
+const fs = require('fs');
+const path = require('path');
+const {google} = require('googleapis');
+
+// Set an interval of time that makes this data stale and converts it in ms
+const FETCH_INTERVAL_HOURS = 4;
+const FETCH_INTERVAL_MILLISECONDS = FETCH_INTERVAL_HOURS * 3600000;
```

Comment:
Since `ms` is a peer dependency already (via `gulp` and a few others), let's install it as a direct dependency and use it here to make things more readable.